tests/unit/botocore/test_utils.py

```diff
@@ -1049,16 +1049,24 @@ def test_is_unaffected_by_sigv4(self):
 
 
 class TestSwitchToChunkedEncodingForNonSeekableObjects(unittest.TestCase):
-    def test_switch_to_chunked_encodeing_for_stream_like_object(self):
+    def test_chunked_encoding_used_for_stream_like_object(self):
```

Comment:
There's nothing technically wrong with this, but we should stay consistent with how the [test was changed in `botocore`](https://github.com/boto/botocore/blob/develop/tests/unit/test_utils.py#L1122-L1139).